src/webgl/p5.RendererGL.js

```diff
@@ -1919,23 +1954,47 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
         defaultShaders.imageLightDiffusedFrag
       );
     }
+    // Create a shader for cubemap conversion
+    const cubemapShader = this._getCubemapShader();
+
+    // Render each face of the cubemap
+    for (let i = 0; i < 6; ++i) {
+      newFramebuffer.draw(() => {
+        cubemapShader.use();
+        cubemapShader.setUniform('equirectangularMap', input);
+        cubemapShader.setUniform('projection', captureProjection);
+        cubemapShader.setUniform('view', captureViews[i]);
+
+        this._pInst.noStroke();
+        this._pInst.rectMode(constants.CENTER);
+        this._pInst.noLights();
+        this._pInst.rect(0, 0, width, height);
+
+        // Capture the rendered face and store it in the faces array
+        faces[i] = this._pInst.get(0, 0, width, height);
```

Comment:
Also probably not the source of the error, but we might need to put this outside of the `draw(...)` block, and use `faces[i] = newFramebuffer.get()` instead of `this._pInst.get(...)`.